src/main/groovy/org/labkey/gradle/plugin/ServerDeploy.groovy

```diff
@@ -364,32 +333,6 @@ class ServerDeploy implements Plugin<Project>
         }
         project.tasks.named('deployApp').configure {mustRunAfter(project.tasks.cleanBuild)}
```

Comment:
I think we can also remove, somewhere close to here, the lines that check for catalinaHome and hasPlugin(Tomcat) since CATALINA_HOME isn't relevant for embedded and we aren't supporting the standalone builds anymore:
```
if (project.plugins.hasPlugin(Tomcat)) {
            if (project.tomcat.hasCatalinaHome()) {
                project.tasks.named("cleanBuild").configure {
                    it.dependsOn(project.tasks.stopTomcat)
                }
                project.tasks.named("cleanDeploy").configure {
                    it.dependsOn(project.tasks.stopTomcat)
                }
            }
        }
```
I think this also cascades to removing the `hasCatalinaHome` method from `TomcatExtension`.